src/Raven.Studio/typescript/components/pages/resources/databases/DatabasesPage.tsx

```diff
@@ -70,6 +77,22 @@ export function DatabasesPage(props: DatabasesPageProps) {
         }
     };
 
+    useEffect(() => {
+        const handleResize = () => {
+            const screenWidth = window.innerWidth;
+            setShowFilterOptions(screenWidth >= 1400);
+            setShowToggleButton(screenWidth < 1400);
+        };
+
+        handleResize();
+
+        window.addEventListener("resize", handleResize);
```

Comment:
do we want to change that on resize?

Reply:
Yes, it prevents the toggle button from being visible in case you move the browser tab from a small screen to a bigger one.